poc/.mypy.ini

```diff
@@ -0,0 +1,6 @@
+[mypy]
+check_untyped_defs = True
```

Comment:
This setting was important, I learned that `mypy` will by default not do any type checking in the bodies of functions/methods with no type hints in their signature. Setting this flushed a lot of little things out.